elfpy/data/postgres.py

```diff
@@ -187,10 +194,51 @@ def add_transactions(transactions: list[Transaction], session: Session):
     try:
         session.commit()
     except sqlalchemy.exc.DataError as err:  # type: ignore
+        session.rollback()
         print(f"{transactions=}")
         raise err
 
 
+def add_user_map(username: str, addresses: list[str], session: Session):
```

Comment:
can you add a `-> None` return type? This is the default, but better to be explicit imo.